Extension/src/LanguageServer/configurations.ts

```diff
@@ -2041,6 +2049,11 @@ export class CppProperties {
                     }
                 }
             }
+
+            if (incorrectExpandedPaths.length > 0) {
+                pathExists = false;
```

Comment:
You could declare `pathExists` as `const` at this location in the file instead of declaring it earlier with a default, incorrect value on line 2004.
```ts
const pathExists: boolean = incorrectExpandedPaths.length === 0;
```